rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf

```diff
@@ -623,7 +623,7 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     tag:'PCI/6.5.2',\
     tag:'paranoia-level/2',\
     ctl:auditLogParts=+E,\
-    ver:'OWASP_CRS/3.3.5',\
+    ver:'OWASP_CRS/3.3.6',\
```

Comment:
This rule above in line 614 uses both `(?i)` in regexp and `t:lowercase` - do we want to care this? (In CRS4 we do not allow this)

Reply:
I don't think we should change that.